litellm/main.py

```diff
@@ -2282,6 +2282,59 @@ def completion(  # type: ignore # noqa: PLR0915
                     original_response=response,
                     additional_args={"headers": headers},
                 )
+        elif custom_llm_provider == "datarobot":
+            api_base = (
```

Comment:
this is not needed, please see how this was done in https://github.com/BerriAI/litellm/pulls?q=is%3Apr+llamafile+is%3Aclosed